libs/product/src/drivers/magento/transforms/simple-product-transformers.ts

```diff
@@ -33,6 +37,17 @@ function getPrice(product: MagentoProduct): number {
 	? product.price_range.maximum_price.regular_price.value : null;
 }
 
+/**
+ * A function for null checking an object.
+ */
+function getDiscountedPrice(product: MagentoProduct): number {
```

Comment:
I see this function copy/pasted several times, can it be extracted?

Reply:
extracted